api/osrd_infra/models/study.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import ArrayField
+
+from osrd_infra.schemas.study import OperationalStudyState, OperationalStudyType
+
+
+class Project(models.Model):
+    name = models.CharField(max_length=128)
+    objectives = models.CharField(max_length=255, null=True)
+    description = models.CharField(max_length=255, null=True)
+    funders = ArrayField(models.CharField(max_length=255), null=True)
+    budget = models.IntegerField(null=True)
+    image = models.ImageField(upload_to="./project_pictures", null=True)
+    studies = ArrayField(models.IntegerField(), null=True)
```

Comment:
You already have a foreign key in `OperationalStudy`.

There is no reason to "double link" your tables.